cmd/genericconf/config.go

```diff
@@ -70,3 +70,36 @@ func ParseLogType(logType string) (log.Format, error) {
 	}
 	return nil, errors.New("invalid log type")
 }
+
+type FileLoggingConfig struct {
+	Enable     bool   `koanf:"enable"`
+	File       string `koanf:"file"`
+	MaxSize    int    `koanf:"max-size"`
+	MaxAge     int    `koanf:"max-age"`
+	MaxBackups int    `koanf:"max-backups"`
+	LocalTime  bool   `koanf:"local-time"`
+	Compress   bool   `koanf:"compress"`
+	BufSize    int    `koanf:"buf-size"`
+}
+
+var DefaultFileLoggingConfig = FileLoggingConfig{
+	Enable:     false,
+	File:       "logs",
+	MaxSize:    100,   // 100Mb
+	MaxAge:     0,     // don't remove old files based on age
+	MaxBackups: 0,     // keep all old files
```

Comment:
If I'm understanding this correctly, MaxSize is the maximum size of a single file and MaxBackups is how many of those to keep. If so, let's set MaxSize to 5 and MaxBackups to 20, because I'm assuming here that every time the node restarts it'll begin a new log file.

Reply:
I made the change, but the file handler behavior is different - after node restart the logs are appended to the previous log file